src/main/CommandHandler.cpp

```diff
@@ -456,6 +457,12 @@ CommandHandler::info(std::string const& params, std::string& retStr)
         info["quorum"] = q["slots"];
     }
 
+    Json::Value invariantFailures = mApp.getInvariantManager().getInformation();
```

Comment:
I think this got moved to a different file in a recent commit, otherwise this is fine